scalableminds/webknossos
# Segment group batch actions

### Steps to test:
- Go to the segments tab in any annotation
- Make sure you see some segments in the list, e.g. by clicking segments
- Either create a new group and drop segments or test the following on the root group:
-- Right-click a group and set the color
-- Revert setting the color
-- Load meshes either ad-hoc or precomputed with a meshfile. If there is no meshfile, there should only be the option to load the meshes ad-hoc
-- Hide/show meshes. The text of the option should be adjusted to the current visibility
-- Download all meshes of the group. You should receive a zip with the according stl files.

### Issues:
- fixes #6954
- Related to #5727

------
(Please delete unneeded items, merge only when none are left open)
